docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -138,13 +169,21 @@ be delayed in order to allow the new release to be shipped as soon as possible.
 
 
 Deprecation Cycle
-`````````````````
+------------------
 
 Sometimes we need to remove a feature, normally in favor of a reimplementation that we hope does a better job.
 To do this we have a deprecation cycle. First we mark a feature as 'deprecated'. This is normally accompanied with warnings
 to the user as to why we deprecated it, what alternatives they should switch to and when (which version) we are scheduled
 to remove the feature permanently.
 
+Ansible deprecation cycle
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since Ansible is a package of individual collections, the deprecation cycle depends on the collection maintainers. We recommend the collection maintainers deprecate a feature in one Ansible major version and do not remove that feature for one year, or at least until the next major Ansible version. For example, deprecate the feature in 2.10.2, and do not remove the feature until 2.11.0.  This is up to each collection maintainer and cannot be guaranteed.
```

Comment:
```suggestion
Since Ansible is a package of individual collections, the deprecation cycle depends on the collection maintainers. We recommend the collection maintainers deprecate a feature in one Ansible major version and do not remove that feature for one year, or at least until the next major Ansible version. For example, deprecate the feature in 2.10.2, and do not remove the feature until 2.12.0.  This is up to each collection maintainer and cannot be guaranteed.
```
Maybe add that collections are supposed to stick to semantic versioning, and since the major version of a collection is fixed per Ansible major version, that a feature deprecated in 2.10.2 will not be removed before 2.11.0 (if collections don't screw it up)?

Reply:
added in the latest commit